### PR TITLE
APICLIENT-26 implement tunews API endpoints

### PR DESCRIPTION
**Pull request by AniesRayyes**

### CODE REVIEW

**Review by mawadkittos — APPROVED** [review]

File: src/endpoints/createLocalNewsElementEndpoint.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import type { JsonLocalNewsType } from '../types'
+import LocalNewsModel from '../models/LocalNewsModel'
+import moment from 'moment-timezone'
+import Endpoint from '../Endpoint'
+
+export const LOCALNEWS_ELEMENT_ENDPOINT_NAME = 'newsElement'
+
+type ParamsType = { city: string, language: string, id: string }
+
+// https://cms-dev.integreat-app.de/testumgebung/en/wp-json/extensions/v3/fcm?id=39
```
Comment (mawadkittos) [inline src/endpoints/createLocalNewsElementEndpoint.js:13]: @orjwan98 Please remove commented code

File: src/endpoints/createLocalNewsEndpoint.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<LocalNewsModel>> =>
     .withMapper((json: Array<JsonLocalNewsType>): Array<LocalNewsModel> => json
       .map((localNews: JsonLocalNewsType) => {
         return new LocalNewsModel({
+	  id: localNews.id,
```
Comment (mawadkittos) [inline src/endpoints/createLocalNewsEndpoint.js:21]: Please fix the indentation here

File: src/types.js [hunk 3]
```diff
@@ -167,6 +177,8 @@ export type JsonCityType = {
   live: boolean,
   events: boolean,
   extras: boolean,
+  tunews: boolean,
+  ["push-notifications"]: boolean,
```
Comment (michael-markl) [inline src/types.js:181]: Phew, that seems pretty inconsistent. We should use an underscore here (just like with `name_without_prefix`).
@klinzo @maxammann Do you agree ?

Comment (maxammann) [reply src/types.js:181]: I agree.
@svenseeberg can we change this?

If yes then I think it should be easy to fix in this PR.

**Review by maxammann — APPROVED** [review]

**Pull request closed without merging**